app/tests/test_asset_upload_model.py

```diff
@@ -78,7 +77,7 @@ def test_create_asset_upload_default(self):
         self.assertEqual(asset_upload.ended, None, msg="Wrong default value")
         self.assertAlmostEqual(
             utc_aware(datetime.utcnow()).timestamp(),
-            asset_upload.created.timestamp(),
+            asset_upload.created.timestamp(),  # pylint: disable=no-member
```

Comment:
Did you verify that `asset_upload.created` really have a `timestamp()` method ?

Reply:
Yes, timestamp seems to still exist. Testcase also passes without any issues.